[intro-herman] Herman-intro blog post

mirisuzanne opened the pull request:
Needs…

- [x] review the text for clarity and typos
- [x] add links
- [x] add images

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+summary: |
+  **Design systems streamline development, communication, and consistency** --
+  but often rely on dedicated teams and extended budgets.
+  We wanted a tool to help us create and maintain
```
Instead of "We" perhaps "OddBird..."

mirisuzanne replied:
Why make this less personal? Is it really not obvious on the OddBird blog who "we" is?

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+  `Herman`_ keeps our development flow simple,
+  and our UX consistent,
+  as we iterate on patterns
+  and scale over time.
```
Mention that Herman is a free, open-source tool.
Perhaps change "our" and "we" to "you" and "your" so people realize this is tool for them to use, not just an internal tool OddBird made for ourselves.

mirisuzanne replied:
To me, using "we" makes it more of a testimonial: this thing has *worked* for us. I think there are better ways to clarify that this tool is available, if that's really not clear yet.

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+===================
+
+I spent the week after Thanksgiving in San Francisco,
+attending (and `accidentally speaking at`_)
```
Don't forget to update your links.

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+
+I'm always curious how many attendees at a conference
+are working on enterprise projects
+over a long period of time,
```
No comma after "time"

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+image:
+  - src: 'blog/2017/herman-intro/header.jpg'
+headline:
+  - tagline: 'An agile approach to design systems'
```
An agile approach to design systems on a budget

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+documenting brand design tokens --
+`colors`_, `fonts`_, `sizes`_, `ratios`_, and `icons`_ --
+before expanding into `practical patterns and components`_,
+and with guides and systems that scale along with us.
```
Again, can you make it clearer that this tool is not just for OddBird, but a free open-source tool for everyone in our position?

mirisuzanne replied:
Can we make that clear once at the top, and then take a personal approach to discussing how we use it?

SondraE commented on content/2017/12/12/introducing-herman.rst:
```diff
@@ -0,0 +1,389 @@
+documentation, and accessibility --
+or help eliminating technical debt,
+and putting better processes in place --
+we're here to help.
```
Link to the training services page somewhere in here.

mirisuzanne commented:
@jgerigmeyer @SondraE I think I've addressed the concerns here (if not the exact suggestions). I also changed the publication date to tomorrow. Let me know if you see any remaining issues.

(the training links will remain broken until merged with the `hermansite` branch)

jgerigmeyer commented on content/2017/12/14/introducing-herman.rst:
```diff
@@ -0,0 +1,417 @@
+
+.. _documentation of CSS selectors and markup patterns: /herman/docs/demo_examples.html
+
+.. image:: /static/images/blog/2017/herman-intro/examples.jpg
```
Where is this screenshot taken from? I don't think it's using the latest version of Herman, because the output HTML isn't being pretty-printed.

mirisuzanne replied:
oh, that was probably on Time Designer. I tried to clean up the output in dev tools, but maybe better to create a better example in the Herman docs or elsewhere?

jgerigmeyer replied:
MCS and OddSite (and Herman itself) are all on the latest version.

mirisuzanne replied:
yeah, Herman doesn't have any nice simple patterns to show this off (though we could create a demo). OddSite doesn't have any documented yet - and broke when I tried (see #173). MedCurbside examples don't seem to use njk yet…

jgerigmeyer commented on content/2017/12/14/introducing-herman.rst:
```diff
@@ -0,0 +1,417 @@
+We're excited to hear from you!
+
+.. _contact form: /contact/
+.. _`@OddBird`: https://twitter.com/oddbird
```
Is there a reason why this link needs backticks around it? Seems to work the same for me either way.

SondraE commented:
@mirisuzanne 
- I like the solution to you came up with to let people know Herman is an open-source tool for them to use!
- I noticed that our headline (and sub headline) pattern for blog posts is to capitalize the first letter of most words (excluding words like the, a, etc). That needs to be updated on Herman blog post.
- What's going on with the top border line on the Icon Previews and Size Previews images? Are those missing on purpose?

mirisuzanne commented:
@SondraE 

- Great!
- All the sub-headlines I see use sentence-case, not title case. Where are you seeing title case?
- I don't know what's going on. I don't see this error either. The borders all look normal on my browsers. Can you paste a screenshot, and details about your browser?

SondraE commented:
@mirisuzanne 
- All the blog posts use title case. Here's a screenshot from Welcome to Susy3! blog post:
![screen shot 2017-12-14 at 12 05 35 pm](https://user-images.githubusercontent.com/8933115/34009730-4040add6-e0c7-11e7-94e7-cadd56843b8a.png)
- I figured out that my zoom was at 90%. When I had it set there, the top borders of those 2 images where disappearing. When I switch to 100%, everything looks as it should. 
![screen shot 2017-12-14 at 12 23 39 pm](https://user-images.githubusercontent.com/8933115/34010358-a94ed6d4-e0c9-11e7-831b-3ba15ae67f22.png)

jgerigmeyer commented:
@SondraE @stacyk Have you both reviewed and approved this? If so, I'll merge it even if we wait to make the post public until Tuesday.

stacyk commented:
I haven't looked yet. I'd say you dont have to wait for me, and we can always make updates if you want to push and we find something. Otherwise, I will do it over the weekend.
